operator/controller/src/main/java/io/apicurio/registry/operator/utils/Mapper.java

```diff
@@ -24,6 +24,14 @@ public class Mapper {
         YAML_MAPPER.configure(FAIL_ON_UNKNOWN_PROPERTIES, true);
     }
 
+    public static <T> T deserialize(String data, Class<T> klass) {
```

Comment:
Similar functionality is already in `io.apicurio.registry.operator.resource.ResourceFactory#deserialize`. Can we a single place for this functionality? I don't have a preference for where to put it.